include/uv-win.h

```diff
@@ -366,8 +366,8 @@ RB_HEAD(uv_timer_tree_s, uv_timer_s);
     struct {                                                                  \
       OVERLAPPED overlapped;                                                  \
       size_t queued_bytes;                                                    \
-    };                                                                        \
-  };                                                                          \
+    } io;                                                                     \
+  } req;                                                                      \
```

Comment:
@TomCrypto doh, you're right, I forgot about that :-S Let's see if someone else has some better name suggestions.